Adding Hierarchical Forecasting

CSampsonSomersetFT opened the pull request.

CSampsonSomersetFT committed:
Adding Hierarchical Forecasting

BenH235 committed:
Add plots/mape/ETS

BenH235 commented:
Made a few very minor changes to the notebook. Included plot of forecasts, added MAPE (discussed in prior notebook), removed benchmark argument from evaluate (normalises accuracy by benchmark, not important either way), and switched ARIMA <-> ETS. I also couldn't get the last cell to run

CSampsonSomersetFT commented:
> Made a few very minor changes to the notebook. Included plot of forecasts, added MAPE (discussed in prior notebook), removed benchmark argument from evaluate (normalises accuracy by benchmark, not important either way), and switched ARIMA <-> ETS. I also couldn't get the last cell to run

Cheers :-)
Which version of hierarchical forecasting are you using? They seem to have changed the interface in the latest version

CSampsonSomersetFT committed:
missed forecast fitting
